README.md

```diff
@@ -468,6 +469,170 @@ ANSWERS.addComponent('DirectAnswer', {
 })
 ```
 
+## Custom Direct Answer Card
+
+You can customize the look and behavior of your DirectAnswer, you can create a custom Direct Answer card.
+
+A custom Direct Answer card is given the directAnswer data from the query, the same data the built-in component
+operates on. That object will look something like the below:
+
+```js
+{
+  type: "FIELD_VALUE",
+  answer: {
+    entityName: "Entity Name",
+    fieldName: "Phone Number",
+    fieldApiName: "mainPhone",
+    value: "+11234567890",
+    fieldType: "phone" 
+  },
+  relatedItem: { 
+    verticalConfigId: 'people',
+    data: { 
+      id: "Employee-2116",
+      type: "ce_person",
+      fieldValues: {
+        description: "This is the description field.",
+        name: "First Last",
+        firstName: "First",
+        lastName: "Last",
+        mainPhone: "+1234567890",
+      }
+    }
+  }
+}
+```
+
+A Direct Answer custom card needs a corresponding template. This can be added either inline by changing the component's constructor to:
+
+```js
+    constructor(config, systemConfig) {
+      super(config, systemConfig);
+      this.setTemplate(`<div> your template here </div>`)
+    }
+```
+
+Or by including a custom template bundle, and adding:
+
+```js
+  static defaultTemplateName () {
+    return 'YourTemplateName';
+  }
+```
+
+Where 'YourTemplateName' is the name of the template registered onto the Handlebars renderer.
+
+The example custom Direct Answer card will use the Handlebars template below.
+
+```hbs
+  <div class="customDirectAnswer">
+    <div class="customDirectAnswer-type">
+      {{type}}
+    </div>
+    <div class="customDirectAnswer-value">
+      {{#each customValue}}
+        {{#if url}}
+          {{> valueLink }}
+        {{else}}
+          {{{this}}}
+        {{/if}}
+      {{/each}}
+    </div>
+  </div>
+
+  {{#*inline 'valueLink'}}
+  <a class="customDirectAnswer-fieldValueLink" href="{{{url}}}"
+    {{#if @root/eventType}}data-eventtype="{{@root/eventType}}"{{/if}}
+    {{#if @root/eventOptions}}data-eventoptions='{{{ json @root/eventOptions }}}'{{/if}}>
+    {{{displayText}}}
+  </a>
+  {{/inline}}
+```
+
+This is an example of a possible custom Direct Answer card. It has custom rendering that depends on
+the fieldType of the directAnswer, as well as a a custom analytics event.
+
+```js
+  class CustomDirectAnswerClassName extends ANSWERS.Component {
+    constructor(config, systemConfig) {
+      // If you need to override the constructor, make sure to call super(config, systemConfig) first.
+      super(config, systemConfig);
```

Comment:
I know setting the template is discussed above the example, but I don't think this example component is setting the template.

Reply:
yeah I talked with Tom about this, he didn't like calling setTemplate inline. @tmeyer2115 ?

Reply:
The reason I suggested not including that explicitly was because there are multiple ways to provide a custom component with a template. I didn't want folks thinking they had to do it that way. Most probably will. I'm fine if you want to add it back in (with a comment indicating this is one of a few ways of supplying the template).

Reply:
sure, added it back with a comment saying this is done for simplicity's sake. we don't provide instructions on how to provide a custom template bundle so I was hesitant to do it like that in the example.